perfaware/sim86/shared/Sim8086.cpp

```diff
@@ -30,16 +30,16 @@ char const* RegisterNames[][3] =
     {"flags", "flags", "flags"}
 };
 
-void SimulateInstruction(instruction Instruction)
+void SimulateInstruction(instruction& Instruction)
```

Comment:
You're not changing Instruction inside the function, right? In that case, it can be a const reference. Same idea as for Dest and Source which you already made const.

Reply:
Correct
e084a8c3a6794c652f7930cbef2bd20d711d3db7